sqs-lambda-ddb-cdk-ts/example-pattern.json

```diff
@@ -0,0 +1,56 @@
+{
+  "title": "Amazon SQS to AWS Lambda to Amazon DynamoDB with CloudWatch Alarms",
+  "description": "Preventing Amazon DynamoDB write throttling with an Amazon SQS queue-based buffer using CDK and monitoring with CloudWatch alarms.",
+  "language": "TypeScript",
+  "level": "200",
+  "framework": "CDK",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This code demonstrates how an Amazon SQS queue can be used to buffer records for an Amazon DynamoDB table and mitigate the risk for a client of being throttled.",
+      "The CDK code deploys Ingress and Dead-Letter SQS queues, AWS Lambda functions, destination DynamoDB tables and CloudWatch alarms.",
+      "AWS Lambda function uses NodeJS 18 runtime."
```

Comment:
Any reason not to use Node v20 as it is the latest supported runtime?

Reply:
updated node version in example.json and in CDK template. updated AWS SDK imports for testing script